src/app/nation/nation.component.html

```diff
@@ -65,3 +67,64 @@ <h3 i18n>Add Nation</h3>
     </div>
   </div>
 </div>
+<!-- Modal for view nations -->
+<div class="modal fade" id="view" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h3 i18n>Nation Details</h3>
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
```

Comment:
we can put title to button with just image. So hovering over could give the idea of what button does